src/templates/addons/versionControl/index.ts

```diff
@@ -0,0 +1,31 @@
+import { GeneralOptions } from '../../../commands/generate';
+import { copy } from '../../../helpers/file';
+
+const versionControlChoices = [
+  {
+    type: 'list',
+    name: 'versionControl',
+    message: 'Enable version control for this project',
+    choices: [
+      {
+        value: 'github',
+        name: 'GitHub',
+      },
+      {
+        value: 'gitlab',
+        name: 'GitLab (NOT IMPLEMENTED YET)',
+        disabled: true,
+      },
```

Comment:
Related to my MEME comment.
IMO we should remove ALL the "not implemented yet" options because they are noise. 
Instead, we can eventually add the issues to GitHub.
If something is not implemented, we should not suggest it as an option.

It feels like 
- "Good morning! Do you want a coffee?"
- "Oh yeah, thanks!"
- "Because the machine is broken!" :harold:

Reply:
Makes sense. I will remove them and create stories for them instead 🙏

Reply:
Removed them in https://github.com/nimblehq/infrastructure-templates/pull/97/commits/becaba37870fac88ad0bb1645d76b9cf8d41b239